cmd/nodecmd/create_gcp.go

```diff
@@ -93,16 +93,24 @@ func getGCPCloudCredentials() (*compute.Service, string, string, error) {
 	return computeService, gcpProjectName, gcpCredentialsPath, err
 }
 
-func getGCPConfig() (*gcpAPI.GcpCloud, map[string]int, string, string, string, error) {
+func getGCPConfig(singleNode bool) (*gcpAPI.GcpCloud, map[string]int, string, string, string, error) {
 	finalRegions := map[string]int{}
 	switch {
 	case len(numNodes) != len(utils.Unique(cmdLineRegion)):
 		return nil, nil, "", "", "", errors.New("number of regions and number of nodes must be equal. Please make sure list of regions is unique")
 	case len(cmdLineRegion) == 0 && len(numNodes) == 0:
 		var err error
-		finalRegions, err = getRegionsNodeNum(constants.GCPCloudService)
-		if err != nil {
-			return nil, nil, "", "", "", err
+		if singleNode {
```

Comment:
no need to this if statement and `getSeparateHostNodeParam` as `getRegionsNodeNum`can be changed and used